packages/react-transform/test/node/index.test.tsx

```diff
@@ -669,6 +669,7 @@ describe("React Signals Babel Transform", () => {
 				};
 			`);
 
+			scope.path.scope.crawl();
```

Comment:
According to https://github.com/babel/babel/issues/8358, the scope bindings are not automatically updated when modifying the ast - this seems to update them